ledger/accountdb.go

```diff
@@ -125,7 +126,7 @@ var accountsResetExprs = []string{
 // details about the content of each of the versions can be found in the upgrade functions upgradeDatabaseSchemaXXXX
 // and their descriptions.
 // TODO - this should be bumped to 6 if we want to enable the new database schema upgrade.
-var accountDBVersion = int32(5)
+var accountDBVersion = int32(6)
```

Comment:
This basically breaks all the tests until we get to them — WDYT of writing new tests or modifying tests that might be working with current v6 support using: 
```
func TestWorkingV6Thing(t *testing.T) {
  accountDBVersion = int32(6)
  t.Cleanup( func() { accountDBVersion = int32(5) } )
  ...
}

Reply:
I think that we should avoid merging this one until the supporting code is done. It would make it much easier.
( I'm going to branch off this one + Pavel's changes and make the corresponding account updates changes. I don't think it would be that complicated )
The change you've proposed above would create a data race, btw.